loopchain/peermanager/peer_manager.py

```diff
@@ -59,8 +60,16 @@ def serialize_as_preps(self) -> list:
                 for peer_id, peer in self._peer_list_data.peer_list.items()]
 
     def load_peers(self) -> None:
-        PeerLoader.load(peer_manager=self)
         blockchain = ObjectManager().channel_service.block_manager.blockchain
+        last_block = blockchain.last_block
+        rep_root_hash = (last_block.header.reps_hash if last_block else
+                         Hash32.fromhex(conf.CHANNEL_OPTION[ChannelProperty().name].get('crep_root_hash')))
+
+        reps: list = blockchain.find_preps_by_roothash(rep_root_hash) or PeerLoader.load()
```

Comment:
64 ~ 68 lines can include in PeerLoader.load(), It can follow Single Responsibility Principle and  PeerLoader unittests will appear all load methods and priority.
But Peerloader will create new relation to ObjectManager. it is not good.
Nevertheless i think it is better.

Reply:
You're right. I applied your precious review.